webapp/src/components/setting.jsx

```diff
@@ -20,24 +21,27 @@ export default class Setting extends React.PureComponent {
             inputId,
             label,
             required,
+            hideRequiredStar,
         } = this.props;
 
         return (
-            <div className='form-group'>
+            <div className='form-group less'>
+                {label &&
```

Comment:
For readability. And add the closing paren

```suggestion
                {label && (
```